db/schema.rb

```diff
@@ -25,6 +25,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "amr_counts", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
make sure you add a unique index on [pipeline_run_id, allele] so as to avoid your earlier issue of having duplicate entries. (See https://github.com/chanzuckerberg/idseq-web/blob/b8e5edde16c9d4d9bfca828870699ea30cafcb8e/db/schema.rb#L331 -- do it using an add_index migration)